src/Expensify.js

```diff
@@ -193,10 +202,12 @@ function Expensify(props) {
                 </>
             )}
 
```

Comment:
Does this mean that we can't initialize the App offline?

Reply:
Ah we flip the flag [here](https://github.com/Expensify/App/blob/522b3123236ef4fac1374eb9a57e23699532f14c/src/libs/actions/Report.js#L1647-L1658) if the app is offline or some of the conditions to check if it's an anonymous user trying to open a report are not met